### DISCUSSION TITLE
cannot find the file 'wazuh-installer.wixobj'

### DISCUSSION THREAD

**Discussion by 69kazu69** [wazuh/wazuh#16562] [Q&A]
I'm trying to run the wazuh-installer-build-msi.bat file but the error ----

light.exe : error LGHT0103 : The system cannot find the file 'wazuh-installer.wixobj -out wazuh-agent-4.4.0-1.msi  -ext WixUtilExtension -ext WixUiExtension' with type 'Source'.

Keeps coming up, can somebody help me with it please....

**Comment by mdiego92** [comment]
Hello @69kazu69,

Did you modify the wxs template file or created new files for the package?

As a workaround you could use the tools in the wazuh-packages repository to simplify the generation of the MSI, for more information about how to use these tools check this page of our documentation:
https://documentation.wazuh.com/current/development/packaging/generate-windows-package.html

Please let me know if you have any remaining questions or issues!
